src/api/AuthServer/AuthServer/Controllers/AuthController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using AuthServer.DTO;
+using AuthServer.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AuthServer.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AuthController : ControllerBase
+    {
+        private readonly AuthService _authService;
+
+        public AuthController(AuthService authService)
+        {
+            _authService = authService;
+        }
+
+        [HttpPost("register")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        public async Task<IActionResult> Register([FromBody]UserRegisterDTO userRegisterDTO)
+        {
+            try
+            {
+                var userDto = await _authService.Register(userRegisterDTO);
+                return Ok(userDto);
+            }
+            catch (InvalidOperationException ex)
+            {
+                return Unauthorized(ex.Message);
+            }
+        }
+
+        [HttpPost("login")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        public async Task<IActionResult> Login([FromBody] UserLoginDTO userLoginDTO)
+        {
+            try
+            {
+                var token = await _authService.Login(userLoginDTO);
+                return Ok(new { Token = token });
+            }
+            catch (InvalidOperationException ex)
+            {
+                return Unauthorized(ex.Message);
+            }
```

Comment:
Good job. Controller without logic and all the logic is in services. Clean code 👍